src/routes/(app)/product/[id]/+page.svelte

```diff
@@ -492,8 +492,17 @@
 										{t('ageWarning.agreement')}
 									</label>
 								{/if}
-								<button class="btn body-cta body-mainCTA" disabled={!acceptRestriction || loading}
-									>{t(`product.cta.${verb}`)}</button
+								<button
+									class="btn body-cta body-mainCTA"
+									disabled={!acceptRestriction ||
+										loading ||
+										(!!data.physicalCartMinAmount &&
+											data.product.price.amount * quantity <=
+												toCurrency(
+													data.product.price.currency,
+													data.physicalCartMinAmount,
+													data.currencies.main
+												))}>{t(`product.cta.${verb}`)}</button
```

Comment:
extract this to a var at least, eg  `{@const cannotOrderPhysicalProduct = ...}` and here
```suggestion
										cannotOrderPhysicalProduct}>{t(`product.cta.${verb}`)}</button
```